.github/workflows/changed_file_trigger_examples_check.yml

```diff
@@ -0,0 +1,128 @@
+name: Test Example
+on:
+  pull_request:
```

Comment:
Add the `paths` field to run only when `examples` folder changes, please refer to the [doc](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore).